test/setup

```diff
@@ -102,49 +104,49 @@ test-exists() {
     if [[ $f =~ ^! ]]; then
       f="${f#!}"
       if [[ $f =~ /$ ]]; then
-        ok "`[ ! -d "$f" ]`" \
+        ok "$([ ! -d "$f" ])" \
           "Directory '$f' does not exist"
       else
-        ok "`[ ! -f "$f" ]`" \
+        ok "$([ ! -f "$f" ])" \
           "File '$f' does not exist"
       fi
     else
       if [[ $f =~ /$ ]]; then
-        ok "`[ -d "$f" ]`" \
+        ok "$([ -d "$f" ])" \
```

Comment:
This is better, but in bash `[[ ... ]]` is prefered to `[ ... ]`.
When you use `[[` you don't need to quote variables inside. 

There is one exception. The RHS of a comparison should be quoted if it has a variable expansion:
```
if [[ $a == "$b" ]]; then ...
```